app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -394,14 +387,33 @@ docker run -d --name kong \
 [Declarative Configuration format]: /gateway/{{page.kong_version}}/reference/db-less-and-declarative-config/#the-declarative-configuration-format
 [Docker Volume]: https://docs.docker.com/storage/volumes/
 
+### Get started with Kong Gateway
+
+Now that you have a running Gateway instance, Kong provides a series of
+[getting started guides](/gateway/{{page.kong_version}}/get-started/comprehensive/)
+to help you set up and enhance your first Service.
+
+If you use the sample `kong.yml` in this guide, you already have a Service and
+a Route configured. Here are a few more things to check out:
+* [Configure a plugin](/gateway/{{page.kong_version}}/get-started/comprehensive/protect-services)
+* [Secure your Services with authentication](/gateway/{{page.kong_version}}/get-started/comprehensive/secure-services)
+* [Load balance traffic across targets](/gateway/{{page.kong_version}}/get-started/comprehensive/load-balancing)
+
+### Clean up containers
+
+If you're done testing Kong Gateway and no longer need the containers, you
+can clean them up using the following commands:
+
+```
+docker kill kong-dbless
+docker kill kong-database
```

Comment:
We shouldn't have a database in `dbless`

Reply:
Ack copy-pasta error